src/ObjImporter.cpp

```diff
@@ -259,17 +259,17 @@ void main(void)
         ramses::GeometryBinding* geometry = m_scene.createGeometryBinding(*effect);
 
         int index_sz = computeIndexCount();
-        const uint16_t *indices = getIndexArray().data();
+        auto indices = getIndexArray();
 
-        const ramses::UInt16Array* rIdxArray = m_client.createConstUInt16Array(index_sz, indices);
+        const ramses::UInt16Array* rIdxArray = m_client.createConstUInt16Array(index_sz, indices.data());
```

Comment:
I assume this error is because the vectors returned by getIndexArray() and getVertexArray() are rvalues - since they are not assigned to anything - and thus are immediately destroyed after the statement, in which case "indices" and "vertexData" would reference invalid memory?

Reply:
That is correct :)